config/dialog/dialog_blacksmith.yml

```diff
@@ -51,49 +51,52 @@ dialogs:
         - quest intro gear
         - flag innkeeper_dislike > 0
         script:
-        - '0 out blacksmith_hello  Hello there, what can i do for you?'
+        - 0 out Hello there, what can i do for you?
         - 5 option reason_dislike
     gear:
         requirements: 
         - quest intro gear
         script:
-        - '0 out blacksmith_hello  Hello there, what can i do for you?'
+        - 0 out Hello there, what can i do for you?
         - 5 option reason
     response_innkeeper_dislike:
         script:
-        - '0 me The innkeeper sent me.'
-        - '20 out blacksmith_innkeeper And I think that means that you are looking for some gear?'
+        - 0 me The innkeeper sent me.
+        - 20 out And I think that means that you are looking for some gear?
         - 0 option looking_dislike   
     response_innkeeper:
         script:
-        - '0 me The innkeeper sent me.'
-        - '20 out blacksmith_innkeeper And I think that means that you are looking for some gear?'
+        - 0 me The innkeeper sent me.
+        - 20 out And I think that means that you are looking for some gear?
         - 0 option looking   
     response_sarcastic_dislike:
         script:
-        - '0 me what else?'
-        - '20 out response_sarcastic_dislike I can see why the Innkeeper doesn''t like you, but I have some gear left over for people such as yourself...'
+        - 0 me what else?
+        - 20 out I can see why the Innkeeper doesn''t like you, but I have some gear left over for people such as yourself...
```

Comment:
Since this line is no longer quoted, you do not need a midline `''` to escape a single quote. `doesn't` will work fine.

Also, "innkeeper" should be lowercase.